### PR TITLE
python311Packages.chromadb: fix by relaxing dependency

### PR DESCRIPTION
**Pull request by GaetanLepage**
## Description of changes

```
Checking runtime dependencies for chromadb-0.5.3-py3-none-any.whl
  - chroma-hnswlib==0.7.3 not satisfied by version 0.7.5
```

cc @fabaff @natsukium 

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 326501` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>10 packages marked as broken and skipped:</summary>
  <ul>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>python311Packages.chromadb</li>
    <li>python311Packages.chromadb.dist</li>
    <li>python311Packages.langchain-chroma</li>
    <li>python311Packages.langchain-chroma.dist</li>
    <li>python312Packages.chromadb</li>
    <li>python312Packages.chromadb.dist</li>
    <li>python312Packages.langchain-chroma</li>
    <li>python312Packages.langchain-chroma.dist</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 326501` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>open-webui</li>
    <li>open-webui.dist</li>
  </ul>
</details>
<details>
  <summary>8 packages built:</summary>
  <ul>
    <li>python311Packages.chromadb</li>
    <li>python311Packages.chromadb.dist</li>
    <li>python311Packages.langchain-chroma</li>
    <li>python311Packages.langchain-chroma.dist</li>
    <li>python312Packages.chromadb</li>
    <li>python312Packages.chromadb.dist</li>
    <li>python312Packages.langchain-chroma</li>
    <li>python312Packages.langchain-chroma.dist</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 326501` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>open-webui</li>
    <li>open-webui.dist</li>
  </ul>
</details>
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li>python312Packages.langchain-chroma</li>
    <li>python312Packages.langchain-chroma.dist</li>
  </ul>
</details>
<details>
  <summary>6 packages built:</summary>
  <ul>
    <li>python311Packages.chromadb</li>
    <li>python311Packages.chromadb.dist</li>
    <li>python311Packages.langchain-chroma</li>
    <li>python311Packages.langchain-chroma.dist</li>
    <li>python312Packages.chromadb</li>
    <li>python312Packages.chromadb.dist</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 326501` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>open-webui</li>
    <li>open-webui.dist</li>
  </ul>
</details>
<details>
  <summary>8 packages built:</summary>
  <ul>
    <li>python311Packages.chromadb</li>
    <li>python311Packages.chromadb.dist</li>
    <li>python311Packages.langchain-chroma</li>
    <li>python311Packages.langchain-chroma.dist</li>
    <li>python312Packages.chromadb</li>
    <li>python312Packages.chromadb.dist</li>
    <li>python312Packages.langchain-chroma</li>
    <li>python312Packages.langchain-chroma.dist</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 326501` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>open-webui</li>
    <li>open-webui.dist</li>
  </ul>
</details>
<details>
  <summary>8 packages built:</summary>
  <ul>
    <li>python311Packages.chromadb</li>
    <li>python311Packages.chromadb.dist</li>
    <li>python311Packages.langchain-chroma</li>
    <li>python311Packages.langchain-chroma.dist</li>
    <li>python312Packages.chromadb</li>
    <li>python312Packages.chromadb.dist</li>
    <li>python312Packages.langchain-chroma</li>
    <li>python312Packages.langchain-chroma.dist</li>
  </ul>
</details>

File: pkgs/development/python-modules/langchain-chroma/default.nix [hunk 1]
```diff
@@ -2,6 +2,7 @@
   lib,
   buildPythonPackage,
   fetchFromGitHub,
+  chroma-hnswlib,
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/langchain-chroma/default.nix:5]: We need to use this somewhere, too.

Comment (GaetanLepage) [reply pkgs/development/python-modules/langchain-chroma/default.nix]: This is not needed in the end.
I have just removed it.

**Review by drupol — APPROVED** [review]

**Comment by drupol** [comment]
@ofborg build python311Packages.chromadb

**Pull request merged**